### PR TITLE
🔀🔖 4.7.9

### PR DESCRIPTION
**Pull request by jxn-30**
**Releasenotes**

* 🐛 [chatExtras/lightDesignChatHistory] fix feature not being loaded
* ♻️ diverse updates in code to improve maintainability
* ✨ [hotkeys] added hotkey for collecting easter eggs, pumpkins etc. (thanks to @aallen92)
* ✨ [shareAlliancePost] add `{{totalDuration}}` variable
* 🇩🇪 Content Pack DE
* 🇳🇱 Content Pack NL
* 🇫🇷 Content Pack FR
* 🇬🇧 Content Pack GB
* 🇺🇸 Content Pack US
* 🇵🇹 First translations for Portugal-Version

---

**Releasenotes for translations**
```md
* 🐛 [$s('chatExtras.lightDesignChatHistory')] fix feature not being loaded
* ♻️ diverse updates in code to improve maintainability
* ✨ [$m('hotkeys')] added hotkey for collecting easter eggs, pumpkins etc. (thanks to @aallen92)
* ✨ [$m('shareAlliancePost')] add `{{totalDuration}}` variable
* 🇩🇪 Content Pack DE
* 🇳🇱 Content Pack NL
* 🇫🇷 Content Pack FR
* 🇬🇧 Content Pack GB
* 🇺🇸 Content Pack US
* 🇵🇹 First translations for Portugal-Version
```

### CODE REVIEW

**Review by Suchty112 — APPROVED** [review]

**Comment by deoxis9001** [comment]
```
* 🐛 [$s('chatExtras.lightDesignChatHistory')]  Correction d'une fonctionnalité qui n'est pas chargée
* ♻️ diverses mises à jour du code pour améliorer la maintenabilité
* ✨ [$m('hotkeys')] ajout d'un raccourci clavier pour collecter les œufs de Pâques, les citrouilles, etc. (merci à @aallen92)
* ✨ [$m('shareAlliancePost')] ajouter la variable `{{totalDuration}}`
* 🇩🇪 Pack de contenu DE
* 🇳🇱 Pack de contenu NL
* 🇫🇷 Pack de contenu FR
* 🇬🇧 Pack de contenu GB
* 🇵🇹  Premières traductions pour la version portugaise
```
fr_FR

**Review by kdev — APPROVED** [review]
TLDR.
LGTM tho!

File: .all-contributorsrc [hunk 1]
```diff
@@ -1,414 +1,346 @@
...
+                "question",
+                "review",
+                "de_DE"
+            ],
+            "login": "jxn-30",
+            "name": "Jan",
+            "profile": "https://github.com/jxn-30"
+        },
+        {
+            "avatar_url": "https://avatars0.githubusercontent.com/u/30264441?v=4",
```
Comment (Cr4zyc4k3) [inline .all-contributorsrc:24]: New linter rule?

File: renovate.json [hunk 2]
```diff
@@ -66,12 +66,14 @@
             "separateMajorMinor": false
         },
         {
+            "assignees": ["jxn-30"],
```
Comment (Cr4zyc4k3) [inline renovate.json:69]: Maybe @Suchty112  as well?

File: src/modules/extendedCallWindow/assets/arrSearch.ts [hunk 3]
```diff
@@ -171,14 +171,14 @@ export default (
         aaoGroupElement.prepend(wrapper);
         import(
             /* webpackChunkName: "modules/extendedCallWindow/components/arrSearchDropdown" */
-            '../components/arrSearch/arrSearchDropdown.vue'
-        ).then(({ default: arrSearchDropdown }) =>
+            '../components/arrSearch/ARRSearchDropdown.vue'
```
Comment (Cr4zyc4k3) [inline src/modules/extendedCallWindow/assets/arrSearch.ts:174]: So, we are using PascalCase instead of camelCase? Should be added to the docs as best practice. And, if available as an eslint rule

File: .all-contributorsrc (hunk 1 above)
Comment (jxn-30) [reply .all-contributorsrc:24]: nopes, that file is not linted at all. allcontributors seem to have changed their output format.

File: src/modules/extendedCallWindow/assets/arrSearch.ts (hunk 3 above)
Comment (jxn-30) [reply src/modules/extendedCallWindow/assets/arrSearch.ts:174]: at least the Vue.js Style Guide strongly recommends PascalCase: https://v2.vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended
but yeah, we can also add it to the docs somewhen.

There is an ESLint Rule but I think we shouldn't enable it before #2479 is completed and all Vue files are transitioned to Composition API. What do you think?

File: renovate.json (hunk 2 above)
Comment (jxn-30) [reply renovate.json:69]: I would like to keep an eye on engine updates myself to be up-to-date about the versions ;)

File: src/modules/extendedCallWindow/assets/arrSearch.ts (hunk 3 above)
Comment (Cr4zyc4k3) [reply src/modules/extendedCallWindow/assets/arrSearch.ts:174]: Add to docs: asap: yeah, but low priority 
Wait till #2479 is completed before enforcing that eslint rule sounds like a good idea as well.

**Review by Cr4zyc4k3 — APPROVED** [review]
